src/Parser/BloodDeathKnight/Modules/Core/BloodMirror.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Module from 'Parser/Core/Module';
+import SPELLS from 'common/SPELLS';
+import ABILITYTRACKER from 'Parser/Core/Modules/AbilityTracker';
+import { calculateMaxCasts } from 'Parser/Core/getCastEfficiency';
+import ISSUE_IMPORTANCE from 'Parser/Core/ISSUE_IMPORTANCE';
+import SpellLink from 'common/SpellLink';
+
+const BLOOD_MIRROR = 120;
+
+class BloodMirror extends Module {
+  static dependencies = {
+    abilityTracker: ABILITYTRACKER,
+  };
+
+    suggestions(when) {
```

Comment:
I think this could also be solved by the CastEfficiency module, why is this separate?

Reply:
Sorry for my bad explanation is the PR lol
Basically this is my goal and reason for separating it. Defensive CDs like this should not be used on CD and shouldnt be encouraged to used on CD like offensive abilities. So i wanted to simply show number of casts without showing the efficiency percent . To do that i had to split it off to make a custom suggestion that shows this. I wanted this to still be on the suggestion page under minor to bring a lil more attention to it instead of it being lost in the list of spells casted. So when people look they know to keep in mind they may need to use them more if needed. I plan on implementing them in the Cooldown tab also to show damage reduced or healing done because of them at a later time.

Reply:
Hello Maybe this pic will show what i want better:

![image](https://user-images.githubusercontent.com/9600587/30999823-79677760-a4a5-11e7-8d97-722558c05719.png)